src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -27,29 +27,39 @@ sub-<label>/[ses-<label>/]
         <matches>[_recording-<label>]_stim.json
 ```
 
+Continuous physiological recordings SHOULD use `_physio.<tsv.gz|json>`
+pairs, for example:
+
+  - pulse measurement,
+  - electrocardiogram,
+  - respiratory movement measured with a respiration belt,
+  - gas concentration,
+  - eye tracking,
+  - head-motion parameters estimated by the MRI scanner.
+
+Continuous signals related to the stimulus SHOULD use the `_stim` suffix.
+
 For the template directory name, `<datatype>` can correspond to any data
 recording modality, for example `func`, `anat`, `dwi`, `meg`, `eeg`, `ieeg`,
 or `beh`.
 
 In the template filenames, the `<matches>` part corresponds to task filename
```

Comment:
Note to self: this should generalize to modalities without task entity. Will update.

Reply:
Done